lib/matplotlib/tests/test_axes.py

```diff
@@ -1749,11 +1749,12 @@ def test_bar_hatches(fig_test, fig_ref):
 
 def test_pandas_minimal_plot(pd):
     # smoke test that series and index objcets do not warn
-    x = pd.Series([1, 2], dtype="float64")
-    plt.plot(x, x)
-    plt.plot(x.index, x)
-    plt.plot(x)
-    plt.plot(x.index)
+    for x in [pd.Series([1, 2], dtype="float64"),
+              pd.Series([1, 2], dtype="Float32")]:
```

Comment:
Is this specifically for "Float32" or would the test also do with "Float64"? If it's only the custom type and precision does not matter, I'd got with "Float64"  to communicate that we're primarily testing the custom pandas type.

Side-note: It seems the capital "Float" types are not yet documented. There's only [a v1.2.0 change note](https://pandas.pydata.org/docs/whatsnew/v1.2.0.html#experimental-nullable-data-types-for-float-data) and the GH issues linked therein.

Reply:
As far as I can tell its undocumented.  Maybe we should hold of on support, or maybe this PR can go in, but without the test?    

I actually disagree with Pandas having a new type here - it seems people want this for pedantic reasons, but I think 99.999% of the world doesn't care if NaN means a failed computation or missing data.  However, if they feel strongly, they should get numpy onboard, and then everyone will have this flag rather than making a new data type.